honeybee_energy/construction.py

```diff
@@ -10,10 +10,10 @@
     EnergyWindowMaterialGasMixture, EnergyWindowMaterialGasCustom
 from .material.shade import _EnergyWindowMaterialShadeBase, EnergyWindowMaterialShade, \
     EnergyWindowMaterialBlind
-
+from .writer import generate_ep_string, parse_ep_string
```

Comment:
should we use `idf` instead of `ep`. It will be helpful specially when `jdf` will also be available so we know it is idf or jdf even though they are both EP format. > `generate_idf_string`, `parse_idf_string`

Reply:
I agree that calling it `idf` is more appropriate since it makes it clearer these workflows do not engage OpenStudio.  So I will make this change now.  However, I don't know with certainty that they are going to change the name of the IDF file extension when it goes to JSON.  If they do change it, it seems likely that they will call it `epJSON` and not `JDF` as noted by this presentation here: http://web.eecs.utk.edu/~jnew1/presentations/2017_IBPSA_JSON.pdf. All of this is just to say that we may need to figure out what to do with the JSON energyplus format when the time comes.